resources/migrations/000_migrations.yaml

```diff
@@ -7781,3 +7781,25 @@ databaseChangeLog:
                   initiallyDeferred: false
                   onDelete: CASCADE
             tableName: pulse_card
+
+  - changeSet:
+      id: 277
+      author: tsmacdonald
+      comment: Added 0.38.0 - Dashboard subscriptions
+      changes:
+        - dropForeignKeyConstraint:
+            baseTableName: pulse_card
+            constraintName: fk_pulse_card_ref_pulse_card_id
+
+  - changeSet:
+      id: 278
+      author: tsmacdonald
+      comment: Added 0.38.0 - Dashboard subscrptions
+      changes:
+        - addForeignKeyConstraint:
+            baseTableName: pulse_card
+            baseColumnNames: dashboard_card_id
+            referencedTableName: report_dashboardcard
+            referencedColumnNames: id
+            constraintName: fk_pulse_card_ref_pulse_card_id
+            onDelete: CASCADE
```

Comment:
Error was:

```
Uncaught exception, not in assertion.: Referential integrity constraint violation: "FK_PULSE_CARD_REF_PULSE_CARD_ID: PUBLIC.PULSE_CARD FOREIGN KEY(DASHBOARD_CARD_ID) REFERENCES PUBLIC.REPORT_DASHBOARDCARD(ID) (129)"; SQL statement:
DELETE FROM "REPORT_CARD" WHERE "ID" = 566 [23503-197]
expected: nil
  actual: org.h2.jdbc.JdbcSQLException: Referential integrity constraint violation: "FK_PULSE_CARD_REF_PULSE_CARD_ID: PUBLIC.PULSE_CARD FOREIGN KEY(DASHBOARD_CARD_ID) REFERENCES PUBLIC.REPORT_DASHBOARDCARD(ID) (129)"; SQL statement:
DELETE FROM "REPORT_CARD" WHERE "ID" = 566 [23503-197]
```

Would extra-appreciate a careful second set of eyes in case I transposed a column name or something